src/api/domain.ts

```diff
@@ -0,0 +1,9 @@
+let domain: string;
+
+if (process.env.NODE_ENV === 'production') {
+    domain = '/';
+} else {
+    domain = 'https://localhost:8080';
```

Comment:
maybe we can move base-url to environmental variable or something similar?

Reply:
Yes, sure, it's just for now. There are a lot of tiny things that I have to keep in mind while set up project. As project will grow, we will remove such code-smell stuff